side-products/merkl.md

```diff
@@ -0,0 +1,60 @@
+# Merkl, by Angle
+
+Merkl is a mechanism to incentivize UniswapV3-type liquidity positions in a flexible and efficient way. It is built and maintained by Angle Labs, but is separate from Angle Protocol.
```

Comment:
We need links to app, and where it's live + placeholder for place of the code of the script